logprep/ng/abc/processor.py

```diff
@@ -105,29 +52,26 @@ class Config(Component.Config):
     __slots__ = [
         "_event",
         "_rule_tree",
-        "result",
         "_bypass_rule_tree",
     ]
 
     rule_class: ClassVar["Type[Rule] | None"] = None
```

Comment:
```suggestion
    rule_class: ClassVar[Type["Rule"] | None] = None
```
I suggest writing the typing with only Rule as string. This way it removes the error unused import warning from the Type import.